app/frontend/components/domains/jurisdictions/jurisdiction-screen.tsx

```diff
@@ -154,7 +161,38 @@ export const JurisdictionScreen = observer(() => {
                   />
                 </Flex>
               </Flex>
-
+              <Flex
+                as="section"
+                direction="column"
+                borderRadius="lg"
+                boxShadow="md"
+                border="1px solid"
+                borderColor="border.light"
+              >
+                <Box
+                  py={3}
+                  px={6}
+                  bg="greys.white"
+                  borderTopRadius="lg"
+                  borderBottom="1px solid"
+                  borderColor="border.light"
+                >
+                  <Heading as="h3" color="theme.blueAlt" fontSize="xl">
+                    {t("jurisdiction.edit.stepCode.title")}
+                  </Heading>
+                </Box>
+                <Flex direction="column" p={6} gap={9}>
+                  <Flex direction="column" gap={2}>
+                    <Text>{t("jurisdiction.edit.stepCode.description")}</Text>
+                    {/* TODO: Add step code help link href */}
+                    {/* <Link href={"#"} isExternal>
+                      {t("jurisdiction.edit.stepCode.helpLinkText")}
+                      <ArrowSquareOut />
+                    </Link> */}
```

Comment:
This TODO is for the future or just forgot to change?

Reply:
this is for the future, we don't have the href yet